projects/core/src/routing/store/reducers/router.reducer.spec.ts

```diff
@@ -199,4 +215,32 @@ describe('Router Reducer', () => {
       cmsRequired: false
     });
   });
+
+  describe('should set correct context for content pages', () => {
+    let context;
+
+    beforeEach(async () => {
+      store.subscribe(routerStore => {
+        context = routerStore.router.state.context;
+      });
+    });
+
+    it('for generic page', async () => {
+      await zone.run(() => router.navigateByUrl('/customCmsPage'));
+      expect(context).toEqual({
+        id: '/customCmsPage',
+        type: PageType.CONTENT_PAGE
+      });
+    });
+
+    it('for route defined with page label', async () => {
+      await zone.run(() => router.navigateByUrl('/cmsPage'));
+      expect(context).toEqual({ id: 'cmsPage', type: PageType.CONTENT_PAGE });
```

Comment:
I would good to have different pageLabel than the url (i.e. `pageLabel: 'testPageLabel`) to show that the page label is taken (not the url like in the generic route)

Reply:
👍